core/controllers/editor.py

```diff
@@ -625,8 +601,8 @@ def get(self, exploration_id, escaped_state_name):
             try:
                 # Normalize the answers.
                 for answer in answers:
-                    answer['value'] = interaction_instance.normalize_answer(
-                        answer['value'])
+                    answer['answer'] = interaction_instance.normalize_answer(
```

Comment:
Perhaps a bit late to say this, but answer['answer'] is kind of awkward. Is it worth renaming to mitigate this?